### ISSUE TITLE
Make jury directions visible during judging

### ISSUE THREAD

**Issue by nikikana** [hatnote/montage#11]
**Future feature request:** Make the directions for the jury more embedded into Montage once judging begins, rather than having them only visible on the first page.  For example, a mouseover or just text at the bottom of the page. Something that makes it possible to refer back to directions without having to exit the round and go back to the main page.

**Comment by mahmoud** [comment]
Agreed. I don't think there's enough emphasis on directions and instructions on the moment. I'll talk with Pawel and see how future versions can improve on this.

**Comment by MichaelNMaggs** [comment]
Agreed that this is needed. Mouseover would work well.
